### ISSUE TITLE
No License Available

### ISSUE THREAD

**Issue by AwaisKazi** [sunra/php-simple-html-dom-parser#51]

**Comment by sunra** [comment]
In composer.json defined:
"license": "MIT"

**Comment by peter279k** [comment]
I think he wants the LICENSE file like [that](https://github.com/peter279k/serviceBundle/blob/master/LICENSE).

**Comment by AwaisKazi** [comment]
Yes, I mean there's no LICENSE file and the information was hard to find.
